doc/user_guide/installation/with-multiple-interpreters.rst

```diff
@@ -0,0 +1,17 @@
+.. _continuous-integration:
+
+Installation with multiple interpreters
+=======================================
+
+Of course it's always possible and safer to analyse on each interpreters.
```

Comment:
Should this be on a new page? I think it fits well in the previous place but I just don't think we should recommend to use the latest interpreter. That's really something that's up to the user's discretion.

Reply:
Hmm well pylint does work better with the latest interpreter because the ast get better all the time. Python 3.5 can't understand f-string, but 3.6 does. So sure the user choose their interpreter but we need to warn them of the consequences.